Externals/ode/include/ode/objects.h

```diff
@@ -34,13 +35,13 @@ extern "C" {
 /**
  * @defgroup world World
  *
- * The world object is a container for rigid bodies and joints. Objects in 
- * different worlds can not interact, for example rigid bodies from two 
+ * The world object is a container for rigid bodies and joints. Objects in
```

Comment:
Let's won't make and commit absolutely unnecessary changes. It clutters both review request and the history, makies it harder to explore the real changes and bugs and makes the 'blame' utility unusable. Revert 'style' changes from this review, please.

Reply:
Style changes includes replacing tabs with spaces, which is necessary in the current OpenXRay codestyle.

I agree with you and disagree with you at the same time.